.github/workflows/push-to-testnet.yml

```diff
@@ -6,15 +6,27 @@ on:
     branches:
       - master
 
+env: 
+  balena-cli: v12.52.1
+
 jobs:
   testnet:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        sbc: [raspi, rockpi]
     steps:
       - name: Checkout
         uses: actions/checkout@v2
+      - name: Change i2c bus for RockPi
+        if: matrix.sbc == 'rockpi'
+        run: |
+          sed -i -E "s/\/dev\/i2c-1:\/dev\/i2c-1/\/dev\/i2c-7:\/dev\/i2c-7/g" docker-compose.yml
+          ROCKPI="-rockpi"
+          echo "ROCKPI=$ROCKPI" >> $GITHUB_ENV
       - name: Balena Deploy
-        uses: nebraltd/balena-cli-action@v12.51.1
+        uses: nebraltd/balena-cli-action@${{ env.balena-cli }}
```

Comment:
i am pretty sure this will fail as I don't think we can pass env variables into uses statement annoyingly

will need to hard code - but at least using matrix strategy it is in much fewer places